dist/amd/ws-header/ws-header.js

```diff
@@ -148,7 +148,7 @@ define(['exports', '../imports', './storage/cookie-storage', './storage/local-st
           _this2.dispatchEvent('ws-auth-changed', accessToken);
         });
 
-        this.constructor.authorization.tryFetchToken(location.hash.substr(1));
+        this.constructor.authorization.tryFetchToken(window.location.hash.substr(1));
```

Comment:
Don't include dist folder in your PR pls. The dist folder and doc folder is auto generated by a circleci process

Reply:
hehe agreed https://github.com/fabric-design/components/pull/158#issuecomment-384986699